ci-operator/step-registry/upi/install/aws/cluster/upi-install-aws-cluster-commands.sh

```diff
@@ -71,7 +70,18 @@ export BOOTSTRAP_URI
 
 # begin bootstrapping
 if openshift-install coreos print-stream-json 2>/tmp/err.txt >coreos.json; then
-  RHCOS_AMI="$(jq -r --arg region "$AWS_REGION" '.architectures.x86_64.images.aws.regions[$region].image' coreos.json)"
+  if [[ "${CLUSTER_TYPE}" == "aws-arm64" ]]; then
+    RHCOS_AMI="$(jq -r --arg region "$AWS_REGION" '.architectures.aarch64.images.aws.regions[$region].image' coreos.json)"
+    BOOTSTRAP_INSTANCE_TYPE=m6g.large
+    MASTER_INSTANCE_TYPE=m6g.xlarge
+    WORKER_INSTANCE_TYPE=m6g.xlarge
+    CLUSTER_TYPE=aws 
+  else
+    RHCOS_AMI="$(jq -r --arg region "$AWS_REGION" '.architectures.x86_64.images.aws.regions[$region].image' coreos.json)"
+    BOOTSTRAP_INSTANCE_TYPE=m5.large
```

Comment:
i think this should be the same default installer uses for x86 that is i3 i guess..you can look it up there